csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -1212,4 +1216,101 @@ void HopperMultipleMatmulScheduler::setUpCircularBuffering() {
   */
 }
 
+void HopperMultipleMatmulScheduler::scheduleStMatrixForMmaOutput(
+    TensorView* tv,
+    int64_t tile_m,
+    int64_t tile_n,
+    int64_t tma_m,
+    int64_t tma_n) {
+  NVF_ERROR(
+      ((tile_m == 16 && tile_n == 16) || (tile_m == 16 && tile_n == 8)),
+      "We only support 16x16 and 16x16 stmatrix now");
+
+  NVF_ERROR(
+      tv->dtype() == DataType::Half, "we only support half type in stmatrix");
+
+  // [M, N] -> [128(TIDx), N/8 ,  2 , 2]
+  auto s =
+      mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(tv->getLoopDomain());
+
+  // Create tma store allocation domain with swizzle
+  scheduleTMAStoreForMmaOutput(tv, tma_m, tma_n);
+
+  tv->setLoopDomain(s.as<IterDomain*>());
+
+  if (tile_m == 16 && tile_n == 16) {
```

Comment:
This is the copied transformations from `mma_utils::scheduleStMatrixForMmaOutput`.